tests/Fixer/LanguageConstruct/ExplicitIndirectVariableFixerTest.php

```diff
@@ -33,29 +33,32 @@ public function testFix(string $expected, ?string $input = null): void
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @return iterable<string, array{0: string, 1?: null|string}>
```

Comment:
```suggestion
     * @return iterable<string, array{0: string, 1?: string}>
```

Reply:
Technically it's possible to return `null` explicitly as a second array item and it will work, because `testFix` signature allows it. I just reflected the actual contract 🙂.

Reply:
it's technically possible to provide null to `testFix`, but I wouldn't like to have provider explicitly returning null.

IMHO we shall have same approaches for all providers, i was always using `1?: string`. if you want to change approach, please change for whole project.

but again, i would avoid provider returning null, either `0: string, 1?: string` or `0: string, 1: null|string`. combining `?` with `|null` is overkill and bringing more than one way to provide data cases

Reply:
I agree, but it's forward-compatible with providers returning more than 2 elements, e.g. rules configuration. I am pretty sure there are already places where input is explicitly returned as `null` because of that.

Reply:
@keradus the example: [signature](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/ab4b588a7e57b898f3775ebeb14915a30bc53289/tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php#L34) + [test case with explicit `null` as the input value](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/ab4b588a7e57b898f3775ebeb14915a30bc53289/tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php#L344-L353). I know it can be changed later if third argument is introduced, but I think it's good to describe existing contract from the beginning.